src/main/java/core/basesyntax/MyHashMap.java

```diff
@@ -1,19 +1,89 @@
 package core.basesyntax;
 
+import java.util.Objects;
+
 public class MyHashMap<K, V> implements MyMap<K, V> {
+    private static final int DEFAULT_CAPACITY = 16;
+    private static final float DEFAULT_LOAD_FACTOR = 0.75f;
+
+    private Node<K, V>[] table;
+    private int size;
+
+    {
+        table = new Node[DEFAULT_CAPACITY];
+    }
+
+    private static class Node<K, V> {
+        private final K key;
+        private V value;
+        private Node<K, V> next;
+
+        Node(K key, V value, Node<K, V> next) {
+            this.key = key;
+            this.value = value;
+            this.next = next;
+        }
+    }
 
     @Override
     public void put(K key, V value) {
-
+        if (size == getThreshold()) {
+            resize();
+        }
```

Comment:
It may be better to create a threshold variable and increase it in resize. And you can delete the getThreshold method and use the threshold instead.